Makefile

```diff
@@ -1,9 +1,17 @@
 $stdout.sync = true
+
+ifdef env
+    ENV_ARG  := $(env)
+else
+    ENV_ARG	 := test
```

Comment:
Super nitpicky, but you may way to align the :=

Reply:
changed, looks like i copied a tab over instead of spaces